eve_glue/wallet_journal_ref.py

```diff
@@ -1,50 +1,8 @@
 """Helpers for wallet journals."""
 
-
```

Comment:
this should be here though https://www.python.org/dev/peps/pep-0008/#blank-lines